piper/build.yaml

```diff
@@ -2,6 +2,7 @@
 build_from:
   amd64: ghcr.io/home-assistant/amd64-base-debian:bullseye
   aarch64: ghcr.io/home-assistant/aarch64-base-debian:bullseye
+  armv7: ghcr.uo/home-assistant/armv7-base-debian:bullseye
```

Comment:
So, upstream support is lacking... how is that fixed by adding these lines?

Reply:
@danijelst You updated the PR by merging master... but you haven't answered this review comment. Maybe you missed it?